lib/tasks/clear_orphans.rake

```diff
@@ -5,7 +5,7 @@ namespace :db do
     OrphanList.destroy_all
     PendingOrphan.destroy_all
     PendingOrphanList.destroy_all
-    Sponsorship.destroy_all
+    Sponsorship.delete_all
```

Comment:
Bypass callbacks, otherwise Sponsorship tries to call `resolve_sponsorship_status` on orphan that has been deleted already.